sqrl-cli/entrypoint.sh

```diff
@@ -88,17 +100,22 @@ fi
 BUILD_UID=$(stat -c '%u' /build)
 BUILD_GID=$(stat -c '%g' /build)
 
-# Pre-create the build directory with proper ownership to avoid permission errors
+# Pre-create the build and logs directories with proper ownership to avoid permission errors
 mkdir -p /build/build 2>/dev/null || true
 chown -R "$BUILD_UID:$BUILD_GID" /build/ 2>/dev/null || true
 
+# Ensure logs directory has proper ownership
+chown -R "$BUILD_UID:$BUILD_GID" /tmp/logs 2>/dev/null || true
```

Comment:
I guess the reason why the logs go under `/tmp` cause it has a 777 permission set and anything can write into it, but that seems a hacky shortcut. :)

Reply:
The part about not writing it under `/build` was clear, but logging under `/tmp` seems a bit odd to me, in most cases the target is under `/var/log` by default. For us that would mean `/var/log/sqrl/...`. But if that adds more complexity, `/tmp` is fine for now I guess, whatever happens, anything that is not mounted will be gone anyways.